sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-full-time-process-integration.scheduler.models.ts

```diff
@@ -0,0 +1,22 @@
+import { OfferingIntensity, RelationshipStatus } from "@sims/sims-db";
+import { MSFAATestInputData } from "./msfaa-helper";
+
+export const MSFAA_FULL_TIME_MARRIED: MSFAATestInputData = {
```

Comment:
For the new tests created, do we need to be in control of this data?
This type of data source alongside the method `saveMSFAATestInputData` was created targeting the file MSFAA file generation where we need to be in control of every single data written to the file.

Reply:
Not using it anymore